lib/article_json/export/amp/elements/embed.rb

```diff
@@ -41,52 +49,28 @@ def type_specific_node
             end
           end
 
-          def youtube_library
-            '<script async custom-element="amp-youtube" ' \
-            'src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>'
-          end
-
           def youtube_node
```

Comment:
a method comment with the return type would be nice :)

Reply:
Yea, I agree – though this is not really part of this PR 😉